### PR TITLE
adding tooltip to dropdown select

### PR DESCRIPTION
**Pull request by bakesaled**
**JIRA**
https://jira.inbcu.com/browse/AUG-811

* This change causes a tooltip to show if the label text is too long to be displayed in the dropdown.
* There will be another PR in audri to rollback previous changes to the dropdown select filter component that added a tooltip.

image

### CODE REVIEW

File: src/lib/dropdown-select/dropdown-select.component.ts [hunk 1]
```diff
@@ -81,6 +91,18 @@ export class DropdownSelectComponent
     this.buildinternalOptions();
   }
 
+  ngAfterViewInit() {
+    // This needs to be async to avoid the ExpressionChangedAfterItHasBeenCheckedError.
+    setTimeout(() => {
+      const labelDiv = this.viewContainerRef.element.nativeElement.querySelector(
+        '.dropdown-label'
+      );
+      console.log(labelDiv.scrollWidth);
```
Comment (craigmichaelmartin) [inline src/lib/dropdown-select/dropdown-select.component.ts]: some console.logs here

File: src/lib/dropdown-select/dropdown-select.component.ts [hunk 2]
```diff
@@ -81,6 +91,16 @@ export class DropdownSelectComponent
     this.buildinternalOptions();
   }
 
+  ngAfterViewInit() {
+    // This needs to be async to avoid the ExpressionChangedAfterItHasBeenCheckedError.
+    setTimeout(() => {
```
Comment (xerona) [inline src/lib/dropdown-select/dropdown-select.component.ts]: Is `setTimeout()` preferred to `detectChanges()` when avoiding making changes in the middle of the lifecycle?

Comment (bakesaled) [reply src/lib/dropdown-select/dropdown-select.component.ts]: no, detectChanges is more efficient.  i'll get that in there.

Comment (xerona) [reply src/lib/dropdown-select/dropdown-select.component.ts]: Thanks, I wasn't sure.

**Pull request merged**